quint/src/parsing/ToIrListener.ts

```diff
@@ -363,43 +367,68 @@ export class ToIrListener implements QuintListener {
     this.declarationStack.push(def)
   }
 
-  // type Alias = set(int)
+  // E.g.,
+  //
+  // - type Alias = Set(int)
+  // - type Constr[a, b] = (Set(a), Set(b))
```

Comment:
This was already wrong before your PR, but here's the fix anyway
```suggestion
  // - type Alias = Set[int]
  // - type Constr[a, b] = (Set[a], Set[b])
```